### DISCUSSION TITLE
`ExtendClientAreaToDecorationsHint` and `TabControl` content

### DISCUSSION THREAD

**Discussion by lostmsu** [AvaloniaUI/Avalonia#12887] [Q&A]
I have a window with `ExtendClientAreaToDecorationsHint` set to `True`, and the entire window is `TabControl`.

The issue is, despite tab item headers not covering the full width of the window, the area to the right from them does not react to attempts to drag the window with mouse.

![image](https://github.com/AvaloniaUI/Avalonia/assets/239520/5d88570a-b290-4309-a434-b001404692a4)

E.g. dragging by the circled zone does not work.

I've tried to explicitly set `TabControl.Background` to transparent to no avail.

This totally does not happen with simple
```xaml
<TextBlock FontSize="24" HorizontalAlignment="Stretch">Hi!</TextBlock>
```

What can be done to get the window draggable by the circled area?

**Comment by timunie** [comment]
Check out FluentAvalonia (https://github.com/amwx/FluentAvalonia/) how they achieved it. Messing with the window border can be a heavy lifting. Basically the background needs to be null or `IsHitTestVisible` must be set to false.
